### ISSUE TITLE
[F2] Research and implement AlgoRai Finance

### ISSUE THREAD

**Issue by ipaleka** [asastats/channel#273]
By @fredestante on [Twitter](https://twitter.com/fredestante/status/1578114824919285767):
> [@asastatscom](https://twitter.com/asastatscom) are you planning to incorporate [@AlgoRai_finance](https://twitter.com/AlgoRai_finance) support?

https://app.algorai.finance/

**Comment by fredestante** [comment]
This is Fred Estante from the Algorand Foundation. If you need direct contact with the AlgoRai team, please let me know.

**Comment by ipaleka** [comment]
> This is Fred Estante from the Algorand Foundation. If you need direct contact with the AlgoRai team, please let me know.

Hi Fred!

First of all, thank you for your steady contribution to the ASA Stats project! People like you are the reason why we have established and why we fully trust in our community contribution system! The primary goal of that system is to keep ASA Stats up-to-date, but also to select the ASA Stats DAO [governors candidates](https://github.com/asastats/channel/wiki/GovernorCandidates). I'm glad to say that you've been added to that list! :) 

We have deployed our first implementation of the AlgoRai Vaults and for now, until we get some feedback, we won't need a direct communication channel with the AlgoRai Finance. But we certainly count on you for such a contact if we encounter problems and/or bugs.
